### PR TITLE
Implemented QFP80 component

### PR DESCRIPTION
**Pull request by abhijitxy**
#39

### CODE REVIEW

File: lib/qfp80.tsx [hunk 1]
```diff
@@ -0,0 +1,107 @@
+import { ChipBody } from "./ChipBody";
```
Comment (seveibar) [inline lib/qfp80.tsx:1]: the file name should just be `qfp`

**Review by seveibar — CHANGES_REQUESTED** [review]
not sure why you introduced a new file in the `lib` dir

**Comment by abhijitxy** [comment]
> not sure why you introduced a new file in the `lib` dir

So they have a different body width

File: lib/qfp.tsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { SmdChipLead } from "./SmdChipLead";
 
 export const QFP = ({
   pinCount,
-  fullWidth = 10,
+  fullWidth = pinCount === 80 ? 14 : 10,
```
Comment (seveibar) [inline lib/qfp.tsx:6]: hmm this is a hack, you should derive the width automatically, we shouldn't hard code width values

**Review by seveibar — CHANGES_REQUESTED** [review]

**Review by seveibar — APPROVED** [review]

**Pull request merged**